doc/manual/local.mk

```diff
@@ -26,9 +26,20 @@ dummy-env = env -i \
 
 nix-eval = $(dummy-env) $(bindir)/nix eval --experimental-features nix-command -I nix/corepkgs=corepkgs --store dummy:// --impure --raw
 
+# re-implement mdBook's include directive to make it usable for terminal output and for proper @docroot@ substitution
+define process-includes
+	while read -r line; do \
+		filename=$$(sed 's/{{#include \(.*\)}}/\1/'<<< $$line); \
+		matchline=$$(sed 's|/|\\/|g' <<< $$line); \
+		sed -i "/$$matchline/r $$(dirname $(2))/$$filename" $(2); \
+		sed -i "s/$$matchline//" $(2); \
+		done < <(grep '{{#include' $(1))
+endef
```

Comment:
Wouldn't this be better/safer as an external script rather than inlined in the makefile?